canonical/ubuntu.com
# Update copy on /kubernetes/compare

## Done

- Updates content based on [copydoc](https://docs.google.com/document/d/1JngYmRhohPo62yU72V1xpzCCILI21SmHKKttBxHjPog/edit)

## QA

- Go to https://ubuntu-com-13890.demos.haus/kubernetes/compare and check the appropriate spelling changes have been made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-11515

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
